src/services/review/RepoCheckerService.ts

```diff
@@ -346,24 +407,99 @@ export default class RepoCheckerService {
         .commit("site checker logs")
         .push(),
       (error) => new SiteCheckerError(`${error}`)
-    )
+    ).map(() => errors)
+  }
+
+  // adapted from https://stackoverflow.com/questions/56427009/how-to-return-papa-parsed-csv-via-promise-async-await
+  csvToRepoError = (repo: string): Promise<RepoError[]> => {
```

Comment:
The name was confusing at first maybe can use something like generateRepoErrorsFromCsv